CHANGELOG.md

```diff
@@ -9,10 +9,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 - Added `Display`, `Error` and `PartialEq` implementations for `Error<E>`
+- Async support based on `embedded-hal-async` 1.0 behind `async` feature flag.
 
 ### Changed
 - [breaking-change] Removed `Default` implementation for `Pca9685` struct.
 - Raised MSRV to 1.62.0.
```

Comment:
```suggestion
- Raised MSRV to 1.75.0.
```
Due to `embedded-hal-async`. Better update it for the whole crate regardless of the feature choice.